CI/JenkinsfileTestCleanInstance

```diff
@@ -58,331 +57,372 @@ pipeline {
+                                    python3 converter/to_junit.py Iptables
+                                """
+                            }
+                        }
+                    }
+                    post {
+                        always {
+                            dir(".") {
+                                junit allowEmptyResults: true, testResults: "tests/output.xml"
+                                archiveArtifacts artifacts: 'tests/test_results*'
+                            }
+                            script {
+                                sh """
+                                    docker stop polycubed || true
+                                    docker system prune --all --force
+                                """
+                            }
+                            sh 'rm -rf ${WORKSPACE}/*'
+                        }
+                        failure {
+                            script {
+                                setGithubStatus("${commit_id}", "error", github_token, "polycube-network/polycube", "Iptables Tests", "Iptables Tests")
+                            }
+                        }
+                        unstable {
+                            script {
+                                setGithubStatus("${commit_id}", "failure", github_token, "polycube-network/polycube", "Iptables Tests", "Iptables Tests")
+                            }
+                        }
+                        success {
+                            script {
+                                setGithubStatus("${commit_id}", "success", github_token, "polycube-network/polycube", "Iptables Tests", "Iptables Tests")
+                            }
+                        }
+                    }
+                }
+            }
         }
-    }
-    }
-    }
-
-
-    stage("Build Docker Mode: pcn-iptables") {
-       agent {
-             label "docker"
-       }
-       steps {
-       script {
-        docker.withRegistry("", 'polycube-repo') {
-        sh """export DOCKER_BUILDKIT=1
-        docker system prune --all --force
-        docker build . -t "polycubebot/${image_name}-pcn-iptables:${image_tag}" --build-arg DEFAULT_MODE=pcn-iptables
-        docker push polycubebot/${image_name}-pcn-iptables:${image_tag}
-        """
+        stage("Promote Images") {
+            when {
+                branch 'master'
+            }
+            agent {
+                label "docker"
+            }
+            steps {
+                script {
+                    docker.withRegistry("", 'polycube-repo') {
+                        sh """
+                            export DOCKER_BUILDKIT=1
+                            docker pull "polycubebot/${image_name}-default:${image_tag}"
+                            docker tag "polycubebot/${image_name}-default:${image_tag}" "polycube-network/polycube:latest"
+                            docker push "polycube-network/polycube:latest"

```

Comment:
The dash between polycube and network should be removed

Reply:
polycubenetwork is the right name

Reply:
Done.